cgmes/cgmes-model/src/main/java/com/powsybl/cgmes/model/triplestore/CgmesModelTripleStore.java

```diff
@@ -127,22 +131,46 @@ public boolean hasBoundary() {
     public boolean isNodeBreaker() {
         // Optimization hint: consider caching the results of the query for model
         // profiles
-        if (queryCatalog.containsKey(MODEL_PROFILES)) {
-            PropertyBags r = namedQuery(MODEL_PROFILES);
-            if (r == null) {
-                return false;
-            }
-            for (PropertyBag m : r) {
-                String p = m.get(PROFILE);
-                if (p != null && p.contains("/EquipmentOperation/")) {
+        if (!queryCatalog.containsKey(MODEL_PROFILES)) {
```

Comment:
If there is no model profiles, shouldn't we throw an exception earlier? For example, when calling `hasEquipmentCore()` instead of returning `false` if `queryCatalog` does not contains `MODEL_PROFILES`?

Reply:
The line you are marking refers to checking if the catalog of queries contains a query that searches the CGMES model for existing profiles. If this query does not exist (for example it does not exist in the CIM14 catalog) it does not mean that we should throw an exception, just that we can not rely on explicit profile information and we have to assume some defaults. 

Small test cases in CIM14 lack these profile references.

Reply:
Ok for me.